### DISCUSSION TITLE
How can i reset binds

### DISCUSSION THREAD

**Discussion by cmkomppa123** [CodenameCrew/CodenameEngine#301] [Q&A] [answered]
Self explanatory

**Comment by r6915ee** [comment]
Go to your AppData folder first if you're on Windows. You can do this by using the key shortcut `Win + R` and then typing in `%appdata%` to open up your `Roaming` AppData folder, which is where Codename and any other HaxeFlixel-related saves go to.

Afterwards, navigate to the folder that is titled `Yoshicrafter29`. Open the `CodenameEngine` folder in there to see a file called `save-default.sol`. You should delete it, which to my knowledge resets your keybinds. Just be aware, however - this will likely delete your entire save data. I am unsure if any other method exists.

**Reply by cmkomppa123** [reply]
Didn't work. infact it was the first thing i tried

**Reply by r6915ee** [reply, marked as answer]
Ah, my bad. After some searching, I found that the file is actually located in the `CodenameEngine` folder that is not in either the `YoshiCrafter29` or `Yoshman29` folder, but instead located right in the `Roaming` folder. There's a file that is titled `options.sol`, and if you delete it your keybinds should be reset. Though this does come with the consequences of your _other_ settings being reset, too, since they are saved in that file as well.

**Reply by cmkomppa123** [reply]
ty
